tests/Tester.py

```diff
@@ -0,0 +1,42 @@
+import csv
+
+from apis import WikipediaApi
+from src import HeuristicTester
+from src.heuristics import Heuristics
+
+test_file_path = "tests.txt"
+results_file_path = "results.tsv"
+
+api = WikipediaApi.WikipediaApi()
```

Comment:
This is ok for a first pass, but we need to be able to configure this between Local and Online apis, right?

Reply:
Yeah I just didn't have the mental energy to get the local api set up on my machine tonight